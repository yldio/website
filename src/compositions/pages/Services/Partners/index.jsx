/* eslint-disable global-require */

import React from 'react';
import styled from 'styled-components';
import remcalc from 'remcalc';

import Container from 'components/Container';
import { H2 } from 'components/Typography';
import breakpoints from 'styles/breakpoints';

import docker from '../../../../assets/docker.png';
import aws from '../../../../assets/aws.png';
import node from '../../../../assets/node.png';

const Bar = styled.div`
  background: #5e5cb2;
  width: ${remcalc(100)};
  height: ${remcalc(4)};
  margin-top: ${remcalc(15)};
  margin-bottom: ${remcalc(70)};
`;

const List = styled.ul`
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 100%;
  flex-direction: column;

  ${breakpoints.tablet`
   flex-directiom: row;
  `};
`;

const Partners = () => (
  <Container>
    <H2 slim>
      Our technology partners
      <Bar />
      <List>
        <li>
          <img src={docker} alt="docker" width="176" height="140" />
        </li>
        <li>
          <img src={aws} alt="aws" width="126" height="75" />
        </li>
        <li>
          <img src={node} alt="node" width="223" height="79" />
        </li>
      </List>
    </H2>
  </Container>
);

export default Partners;
