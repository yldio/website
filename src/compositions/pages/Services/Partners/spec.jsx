import React from 'react';
import { shallow } from 'enzyme';
import Partners from './index';

test('compositions/pages/Services/Partners snapshot', () => {
  const wrapper = shallow(<Partners />);

  expect(wrapper).toMatchSnapshot();
});
