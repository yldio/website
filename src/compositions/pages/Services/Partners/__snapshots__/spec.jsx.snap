// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services/Partners snapshot 1`] = `
<Container>
  <Typography__H2
    slim={true}
  >
    Our technology partners
    <Partners__Bar />
    <Partners__List>
      <li>
        <img
          alt="docker"
          height="140"
          src="/src/assets/docker.png"
          width="176"
        />
      </li>
      <li>
        <img
          alt="aws"
          height="75"
          src="/src/assets/aws.png"
          width="126"
        />
      </li>
      <li>
        <img
          alt="node"
          height="79"
          src="/src/assets/node.png"
          width="223"
        />
      </li>
    </Partners__List>
  </Typography__H2>
</Container>
`;
